app/assets/javascripts/discourse/controllers/insert-hyperlink.js.es6

```diff
@@ -0,0 +1,47 @@
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+import { default as computed } from "ember-addons/ember-computed-decorators";
```

Comment:
ESLint error, `computed` is no longer used: https://travis-ci.org/discourse/discourse/jobs/593993307#L701-L702